api/CreateUserScript.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const User = require('./models/User.js');
+const inquirer = require('inquirer');
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+
+console.log('Welcome to the SCE User Creation CLI!');
+inquirer
+  .prompt([
+    {
+      type: 'input',
+      name: 'email',
+      message: 'What\'s your email:'
+    },
+    {
+      type: 'input',
+      name: 'firstName',
+      message: 'Please enter a first name for your account (default: "Dummy"):'
+    },
+    {
+      type: 'input',
+      name: 'lastName',
+      message: 'Please enter a last name for your account (defualt: "Account"):'
+    },
+    {
+      type: 'input',
+      name: 'password',
+      message: 'Please enter a password (default: "sce"):'
+    },
+    {
+      type: 'list',
+      name: 'accountLevel',
+      message: 'Choose an account level from the list below: ',
+      choices: [
```

Comment:
We have all of the [enums in src](https://github.com/SCE-Development/Core-v4/blob/c07c47f7ca565c00d6eee21f52a7b79cb487b90a/src/Enums.js#L34)

Do you think you could parse that list instead? it will dynamically respond to enum changes in the future.

Reply:
thoughts @evanugarte ?

Reply:
that's a good idea actually, i'll pull that in and use those values instead.